### DISCUSSION TITLE
Als Schüler will ich das meine Schule automatisch als Ziel und meine Standort als Startpunkt ausgewählt wird für die Berechnung des Schulweges, falls ich das will, um Zeit zu sparen.

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#19] [User Stories]
### Beschreibung:
Es soll eine Anbindung der Google Maps oder Deutsche Bahn API geben, die es Schülern ermöglicht einfach zu sehen wie sie heute am schnellsten oder generell zur Schule kommen. Am besten gibt es eine "kleine" Ansicht welche alles zusammenfasst und beim drauf klicken werden mehr Informationen angezeigt oder man wird zur Seite (Google Maps oder DB) weitergeleitet mit den angegeben Ziel und Startpunkt.

Risiken : Externe Probleme mit der Bereitstellung der API

### Akzeptanz-Kriterien: 
 - [ ] API Anbindung wird auf unsere Website angezeigt 
 - [ ] Man kann Ziel und Startpunkt festlegen
 - [ ] Bei darauf klicken wird man auf die orginale Seite weiter geleitet

**Extras:**
- [ ] Der Schulstandort wird automatisch vorausgewählt
- [ ] Der User kann wenn er will seinen Standort automatisch übergeben (auf Anfrage)

**Comment by Schmarvinius** [comment]
Kontext noch dazu schreiben :)

**Comment by two-Gee** [comment]
nicht mehr in requirements
